doc/scheduler-scripting.tex

```diff
@@ -563,9 +563,10 @@ \subsubsection{Graphical Applications}
 \end{verbatim}
 \normalsize
 
-\begin{figure}[htpb]
-	\includegraphics[width=\columnwidth]{images/pycharm}
-    \caption{PyCharm Starting up on a Speed Node}
+\begin{figure}[h]
+	\centering
+		\includegraphics[width=0.85\textwidth]{images/pycharm}
```

Comment:
I  suggest keeping `\columnwidth` the way it was.

Reply:
fixed